SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
```

Comment:
I like using this link because it pulls in other efforts like go-vela 👍

Reply:
Agreed